components/src/dynamo/planner/monitoring/traffic_metrics.py

```
@@ -371,6 +377,85 @@ def warn_if_router_not_scraped(self) -> None:
+        ``<dgd_name>-<replica-index>-<service-key-lowercase>-<hash>``
+        (e.g. ``qwen3-quickstart-0-vllmworker-86nvj``).
+        """
+        try:
+            result = self.prom.custom_query(
+                f"sum(DCGM_FI_DEV_POWER_USAGE{{"
+                f'exported_namespace="{k8s_namespace}",'
+                f'exported_pod=~"^{dgd_name}-[0-9]+-.*"}})'
+            )
+            if result:
+                return float(result[0]["value"][1])
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)